tests/test_shadow_run_id.py

```
@@ -0,0 +1,180 @@
+        assert {o["run_id"] for o in b.get_all_orders()} == {"shadow-bbbbbbbbbbbb"}
+
+    def test_a_registry_without_an_id_still_uses_the_process_one(
+            self, tmp_path, monkeypatch):
+        from core_brain import order_registry
+        from core_brain.order_registry import OrderRegistry, OrderRecord
+
+        monkeypatch.setattr(order_registry, "_CURRENT_RUN_ID", "run-liveone")
+        reg = OrderRegistry(db_path=tmp_path / "c.db")
+        reg.create_order(OrderRecord(
+            id="local-1", order_id="oid-1", condition_id="0xabc",
+            token_id="tok", side="BUY", price=0.47, original_size=20,
```

Ruff (0.16.2)

[error] Possible hardcoded password assigned to argument: "token_id"

(S106)